docs/keycloak/manual-client-secret-setup.md

```
@@ -1,5 +1,7 @@
 ### 📘 This guide is also available in the [🇵🇱 Polish version](manual-client-secret-setup-pl.md)
 
+## To run the application that communicates with a remote Keycloak instance, you need to obtain the current access credentials and update the .env file located in the root directory of the project accordingly. Then, simply run the application in the dev environment.
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)